build_tools/sdk.py

```diff
@@ -154,6 +157,42 @@ def get_local_darwin_sdk_version(platform):
     return run.shell_command('xcrun -f --sdk %s --show-sdk-platform-version' % _convert_darwin_platform(platform)).strip()
 
 
+## **********************************************************************************************
+## Android
+
+
+def get_android_local_sdk_path():
+    path = os.environ.get('ANDROID_HOME', None)
+
+    if path is None:
+        if sys.platform == 'darwin':
+            path = os.path.expanduser('~/Library/android/sdk')
```

Comment:
Not sure yet what the default folders are for Windows/Linux.
Doesn't matter too much, as ANDROID_HOME is used first.